docs/api/zh/math/Quaternion.html

```diff
@@ -44,6 +44,13 @@ <h3>[name]( [param:Float x], [param:Float y], [param:Float z], [param:Float w] )
 
 		<h2>Properties</h2>
 
+		<h3>[property:Boolean isQuaternion]</h3>
+		<p>
+			Used to check whether this or derived classes are Quaternions. Default is *true*.<br /><br />
```

Comment:
We usually use `TODO` as a placeholder so a Chinese community member can easily find missing parts and translate them.

Reply:
Ok, I can change that, I assumed it was needed to be in english at least, viewing that the entire file is in english as well.

Reply:
It can even be in chinese directly in fact, if you validate the other translation.

Reply:
Oh, I did not see in the preview that the rest of the file is not yet translated. You can leave the text as it is 😉